code/modules/research/rdmachines.dm

```diff
@@ -47,7 +47,7 @@
 
 
 
-/obj/machinery/r_n_d/attackby(obj/item/O, mob/user, params)
+/obj/machinery/r_n_d/attackby(obj/O, mob/user, params)

```

Comment:
Ah right, that was when I was originally busy changing the defines the first time, forgot to change them back.